src/MeetingsSDKAdapter.js

```diff
@@ -51,6 +52,7 @@ const EVENT_REMOTE_SHARE_START = 'meeting:startedSharingRemote';
 const EVENT_REMOTE_SHARE_STOP = 'meeting:stoppedSharingRemote';
 
 // Adapter Events
+const EVENT_MEETING_UPDATED = 'meeting:update';
```

Comment:
`'adapter:meeting:updated'`

Reply:
Modification made and moved to https://github.com/webex/sdk-component-adapter/pull/158/files#diff-67d6f2edd2655b86027c2d6d69ae37ba5e655b9f49c43d042f86a2c8c2b981faR54